base/test.jl

```diff
@@ -160,7 +160,7 @@ macro test(ex)
     if isa(ex, Expr) && ex.head == :comparison
         # Generate a temporary for every term in the expression
         n = length(ex.args)
-        terms = [GenSym(i) for i in 1:n]
+        terms = [symbol(:_term_,i) for i in 1:n]
```

Comment:
does this have any measurable impact on https://github.com/JuliaLang/julia/issues/15346